clio-integration-test/src/test/resources/docker-compose/docker-compose-jenkins.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  clio-jenkins-integration-test:
+    image: broadinstitute/clio-integration-test:${CLIO_DOCKER_TAG}
+    environment:
+      ELASTICSEARCH_HTTP_HOST: elasticsearch1.gotc-dev.broadinstitute.org
+    network_mode: "host"
+  clio-server:
+    image: broadinstitute/clio-server:${CLIO_DOCKER_TAG}
```

Comment:
Rather than declaring all the parameters for the server container here in addition to in the other docker-compose file, could we bust out the definition to a separate YAML and then import it here [like we do for elasticsearch](https://github.com/broadinstitute/clio/blob/develop/clio-integration-test/src/test/resources/docker-compose/docker-compose.yml#L15)? Same for the integration-test-runner container too.